mbed_host_tests/host_tests_plugins/module_copy_mps2.py

```diff
@@ -17,96 +17,91 @@
 Author: Przemyslaw Wirkus <Przemyslaw.Wirkus@arm.com>
 """
 
-import re
-from os.path import join
+import os
+from shutil import copy
 from host_test_plugins import HostTestPluginBase
 
 
 class HostTestPluginCopyMethod_MPS2(HostTestPluginBase):
-    # MPS2 specific flashing / binary setup funcitons
+    """ Generic flashing method for mbed-enabled devices (by copy)
+    """
```

Comment:
Lets fix this comment, seems like the older comment was better (seeing how it _is_ a specific flashing function for the MPS2)